frontend/next.config.js

```diff
@@ -10,9 +10,17 @@ const isProdBuild = process.env.NODE_ENV === nodeEnv.PRODUCTION;
 
 const PLAUSIBLE_URL = "https://plausible.io";
 
-const TWITTER_URL = "https://cdn.syndication.twimg.com platform.twitter.com";
+const TWITTER_URL = ["https://cdn.syndication.twimg.com platform.twitter.com"];
```

Comment:
@sethfeingold what's the reason behind changing this link to an array? Thanks!

Reply:
Ahh, can revert that! I think I was originally playing around locally with creating an array of url's (wistia, twitter, and plausible) and passing them to SCRIPT_SRC array, but chose to go back to keeping url's as separately declared strings. Left the brackets behind.

Reply:
Got it that makes sense! Thanks for reverting 🙌 LGTM!!